{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -7,13 +7,23 @@ ARG SECRET_KEY
 
 # libpq-dev and python3-dev help with psycopg2
 RUN apt-get update \
-  && apt-get install -y python3.7-dev python3-pip libpq-dev \
+  && apt-get install -y python3.7-dev python3-pip libpq-dev curl \
   && apt-get clean all \
   && rm -rf /var/lib/apt/lists/*
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash \
```

Comment:
Instead of including all of this inside the custom Dockerfile we have for Python, consider just adding the node image to the docker-compose.yml and running it as a separate container that speaks with the django container. And don't build production code, use `npm start` just like you would in development. The point of docker-compose.yml in this project isn't to deploy stuff, but to make zero-configuration development possible locally.

Here are instructions on how to include this https://github.com/nodejs/docker-node/blob/master/README.md#how-to-use-this-image, particularly the `If you prefer Docker Compose:` section